scraper/manual/manual_urls.py

```diff
@@ -0,0 +1,21 @@
+import csv
+from typing import Iterator
+
+
+from utils.vmd_logger import get_logger
+
+
+logger = get_logger()
+
+
+def manual_urls_iterator() -> Iterator[dict]:
+    logger.info("Recherche des urls manuel")
+    path_file = "data/input/manual_urls.csv"
+    total = 0
+    with open(path_file, encoding="utf8") as csvfile:
```

Comment:
Je ne suis pas sûr de ce que tu cherches à faire ici. `for row in reader` va épuiser l’itérateur du CSV, donc `yield row` va seulement donner la dernière ligne du CSV.

Pour moi, soit tu retournes un intégrateur, soit tu comptes. Si tu comptes, ça veut dire que soit :
* tu as pu tout charger en mémoire -> pas besoin d’intégrateur
* tu n’as pas pu tout charger en mémoire -> il faudrait itérer une deuxième fois

Reply:
J'ai testé avec un centre maiia:
u2;CENTRE DE VACCINATION - toto;;0.99999998144779;50203_0130;;rue de toto;72336;72336;SABLE SUR SARTHE;;;;;;;;;;;;;2021-05-13 14:29:49.010388;;;;;;;;;;;18/01/2021;https://www.maiia.com/centre-de-vaccination/72170-beaumont-sur-sarthe/centre-de-vaccination---beaumont-sur-sarthe?centerid=603cca7668359b1abff14c7b;33243624737;;;;;;;
```

résultat dans le 72.json : 

"centres_disponibles": [
    {
      "departement": "72",
      "nom": "CENTRE DE VACCINATION - toto",
      "url": "https://www.maiia.com/centre-de-vaccination/72170-beaumont-sur-sarthe/centre-de-vaccination---beaumont-sur-sarthe?centerid=603cca7668359b1abff14c7b",
      "location": null,
      "metadata": {
        "address": " rue de toto, 72336 SABLE SUR SARTHE",
        "phone_number": "+3333243624737",
        "business_hours": {
          "lundi": "",
          "mardi": "",
          "mercredi": "",
          "jeudi": "",
          "vendredi": "",
          "samedi": "",
          "dimanche": ""
        }
      },
      "prochain_rdv": "2021-05-17T14:50:00+00:00",
      "plateforme": "Maiia",
      "type": "vaccination-center",
      "appointment_count": 100,
      "internal_id": "maiia603cca7668359b1abff14c7b",
      "vaccine_type": [
        "ARNm"
      ],
      "appointment_by_phone_only": false,
      "erreur": null,
      "last_scan_with_availabilities": "2021-05-13T22:55:36.772303+02:00",
      "appointment_schedules": [
        {
          "name": "chronodose",
          "from": "2021-05-13T00:00:00+02:00",
          "to": "2021-05-14T23:59:59+02:00",
          "total": 0
        },
        {
          "name": "1_days",
          "from": "2021-05-13T00:00:00+02:00",
          "to": "2021-05-13T23:59:59+02:00",
          "total": 0
        },
        {
          "name": "2_days",
          "from": "2021-05-13T00:00:00+02:00",
          "to": "2021-05-14T23:59:59+02:00",
          "total": 0
        },
        {
          "name": "7_days",
          "from": "2021-05-13T00:00:00+02:00",
          "to": "2021-05-19T23:59:59+02:00",
          "total": 100
        },
        {
          "name": "28_days",
          "from": "2021-05-13T00:00:00+02:00",
          "to": "2021-06-09T23:59:59+02:00",
          "total": 100
        },
        {
          "name": "49_days",
          "from": "2021-05-13T00:00:00+02:00",
          "to": "2021-06-30T23:59:59+02:00",
          "total": 100
        }
      ],
      "gid": "u2"
    }
  ],
```
pour le u1 u1;Centre de vaccination Covid-19 - Saint-Pierre-et-Miquelon;;0.99999998144779;50203_0130;;Rue Victor Briand;97500;97502;Miquelon;47.1025473;-1.61351;130007909;8412Z;;;;;;;;;2021-05-13 14:29:49.010388;;;;;;;;;t;;18/01/2021;https://www.doctolib.fr/centre-de-vaccinations-internationales/saint-pierre-et-miquelon/centre-de-vaccination-covid-19-st-pierre-miquelon?pid=practice-176716;33508411690;;;;;;;{}

Surement un pb de lien ou de code insee ? je ne sais pas. mais visiblement ma PR fonctionne

Reply:
Etant clairement débutant sur python, l'idée d'utiliser un sheet et bien. Par contre hors de mes compétences actuelle. Soit vous me laissez clairement du temps, soit quelqu'un peut prendre ma PR (en me le notifiant ) pour éviter que je perde mon temps. 
merci
daien57 sur mattermost